README.rst

```diff
@@ -88,6 +88,8 @@ pydocstringformatter_ (automated pep257).
 Install
 -------
 
+.. This is a copy paste of 'How do I install Pylint?' in the FAQ do not modify one without modifying the other
```

Comment:
Can't we just use `Sphinx` to include one file in both places? That should be possible right?

Reply:
We can't include files in the readme because of github limitations that won't evolve ([security reasons](https://github.com/github/markup/issues/172#issuecomment-33241601)). SO I think it's the only way but if you have a way to do it I'll be glad to create a file.

Reply:
See the documentation that is linked there:
https://docutils.sourceforge.io/docs/ref/rst/directives.html#including-an-external-document-fragment

Can't we include from `README.rst` and use line number or `start-after` based on the headings?

Reply:
Nice, let me try !

Reply:
It works, but any change to the README will trip up the line number. Probably better to keep the comment and copy pasting, it's less underhanded / error prone imo.

Reply:
Did you also try `start-after`? It seems like that should work with our headings?